src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1340,7 +1372,16 @@ class ongoingTasksStats extends viewModelBase {
                 const batchPerf = batchPerformance[perfIdx];
 
                 const perfWithCache = batchPerf as SubscriptionBatchPerformanceStatsWithCache;
+
+                const connection = subscriptionItem.ConnectionPerformance.find(x => x.ConnectionId === perfWithCache.ConnectionId);
                 
+                if (!connection) {
+                    continue;
+                }
+                
+                const workerId = connection.WorkerId;
+                const workerIndex = workersArray.findIndex(x => x === workerId);
```

Comment:
same here - it would be nice to reuse that info

Reply:
done